.github/workflows/_review-code.yml

```
@@ -157,6 +157,10 @@ jobs:
           anthropic_api_key: ${{ steps.get-kv-secrets.outputs.ANTHROPIC-CODE-REVIEW-API-KEY }}
           track_progress: true
           use_sticky_comment: true
+          plugin_marketplaces: |
+            https://github.com/bitwarden/ai-plugins
```

🤔 I have researched the marketplace setup in other repos and I have seen the marketplace references most often with the `.git` extension. I don't know if it matters or not but wanted to mention.

```suggestion
            https://github.com/bitwarden/ai-plugins.git
```